sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/plan/PlanUtils.java

```diff
@@ -30,9 +30,9 @@ private PlanUtils() {
     }
 
     public static final Collection<? extends Step> getCandidates(Plan plan, Collection<String> dirtyAssets) {
-        Collection<Phase> candidatePhases = plan.getStrategy().getCandidates(plan, dirtyAssets);
+        Collection<Phase> candidatePhases = plan.getStrategy().getCandidates(plan.getChildren(), dirtyAssets);
```

Comment:
Yes, via `DependencyStrategyHelper`. Added an interrupted parameter that the `Strategy`s have to pass.